src/actions.js

```diff
@@ -37,7 +37,7 @@ const actions = {
     setPuzzle(pid, puzzle);
   },
   // puzzle: { title, type, grid, clues }
-  createPuzzle: (puzzle, cbk) => {
+  createPuzzle: async (puzzle, cbk) => {
```

Comment:
i think this is unnecessary

Reply:
Gotcha. IntelliJ was yelling at me for ignoring the Promise returned from `db.ref('counters').transaction()` , but I guess since we're not doing anything with it anyways, I guess we don't need it. I updated the PR to remove the unnecessary async/await.